Sources/Plasma/PubUtilLib/plNetClientComm/plNetClientComm.h

```diff
@@ -76,6 +76,11 @@ struct NetCommPlayer {
     plString    playerName;
     plString    avatarDatasetName;
     unsigned    explorer;
+
+    NetCommPlayer() { }
+    NetCommPlayer(unsigned id, const plString& name, const plString& shape, unsigned ex)
+        : playerInt(id), playerName(name), avatarDatasetName(shape), explorer(ex)
```

Comment:
The `emplace_back` probably would work with a braced initializer list without the constructor, I think...  Not that having a constructor is bad, just FYI ;)